hack/externalnode/install-vm.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 [--ns <NameSpace>] [--bin <AntreaAgentSavePath>] [--config <AgentConfigSavePath>] [--kubeconfig-dir <KubeconfigSavePath>] [--help|-h]
+Generate a YAML manifest for Antrea using Helm and print it to stdout.
+        --ns                          NameSpace to be used by the antrea-agent.
+        --bin                         Path of the antrea-agent binary
+        --config                      Path of the antrea-agent configuration file
+        --kubeconfig                  Path of the kubeconfig to access K8s API Server
+        --antrea-kubeconfig           Path of the kubeconfig to access Antrea API Server
+        --help, -h                    Print this message and exit"
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+AGENT_BIN_PATH=""
+CONFIG_PATH=""
+KUBECONFIG=""
+ANTREAKUBECONFIG=""
+AGENT_NAMESPACE=""
+AGENT_LOG_DIR="/var/log/antrea"
+AGENT_CONF_PATH="/etc/antrea"
+
+check_supported_platform() {
+  echo "Checking platform supported"
+  if [[ $(lsb_release -rs) != "20.04" ]]; then
```

Comment:
Do we only support Ubuntu 20.04?

Reply:
@wenyingd 
I have earlier tried with Ubuntu 18.04 and 20.04. But  I have not tested it other version of Ubuntu. 
Do we want to claim official support only for one particular version of Ubuntu for the first release?

Reply:
I don't think we should limit the supported versions on 20.04 only since 18.04 is actually working well. Actually antrea CI testbed is using Ubuntu 18.04.

Reply:
Added check to support 18.04/20.04.